spec.md

```diff
@@ -1422,3 +1442,21 @@ If the Platform encounters an internal error provisioning a Service Instance or
 Service Binding (for example, saving to the database fails), then it MUST at
 least send a single delete or unbind request to the Service Broker to prevent
 the creation of an orphan.
+
+## Extensions
+
+A Service Broker might want to extend the Service Broker API to include custom endpoints specific to their service. An example of this could be lifecycle management, (e.g. "Day Two Operations"), like Backup, Restore, Stop, Start, Restart and Pause. Extending the Service Broker API can be done on a per Service Instance basis, during provisioning. The provisioning request will return a URI to an OpenAPI 3.0+ document that the Platform can use to determine the new endpoint(s), parameter(s), authentication mechanism and server URL. See [Provisioning](#provisioning) for more information on extensions on a per Service Instance basis.
+
+### OpenAPI Document
+
+To extend the API, the Service Broker MUST create an OpenAPI 3.0+ document describing the additional API endpoints and parameters. The returned OpenAPI document MUST be in json format. See the [OpenAPI Specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md) for more information.
+
+### Extension API on Remote Servers
+
+Additional API endpoints MAY be executed on a remote server, however the OpenAPI document MUST include the servers `url` so that the Platform will know how to invoke the endpoint(s). If the server `context` parameter of the OpenAPI document is set to local host the Platform can assume the extension API endpoints are to be invoked using the Service Broker host and port.
```

Comment:
s/can assume.../MUST use.... 

I assume that its just the host (localhost) that is swapped out in favor of the original host/port, right? meaning any path stuff in the OpenAPI doc should still be used right?  We should probably say that if so.  You may want to also say that any port number specified will be replace too if "localhost" is used - if that's your intent.

Reply:
I'm talking specifically about the server open, https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md#serverObject, url. It can or cannot include a port. I'm think we want to say if it just says http://localhost that signifies that it should run on the SB host/port. However, there is an issue if the OAPI doc provides more than one url. Perhaps we can say it provides a url with a description "Service Broker host".

I need to think about this more, so I didn't modify it. Can get to it tomorrow.